### ISSUE TITLE
Changes to Clock Scrambles/Regulations

### ISSUE THREAD

**Issue by CoreySakowski** [thewca/wca-regulations#382]
A couple of proposals to make clock run more easily (more specifically, to avoid problems with the puzzle falling over while the cover is being lifted, and thus having to rescramble it):

1) For all clock scrambles, the pins will all be either up or down.
This would make the event easier to run, as the clock can be placed flat on the surface prior to inspection, and there would be no fear of it falling over. We would only have to make sure the clock is actually facing up the correct way.

2) Pins can be altered during inspection, and the position of the pins prior to inspection does not matter.
This would make things even easier. It is worth noting that pin positions do not affect whether the puzzle is solved, so by that logic, changing a pin does not affect the scrambled state of a puzzle either. But of course, this would shave off a significant amount of time at the beginning of the solve.

**Comment by timreyn** [comment]
I agree with (1).  I think that pin positions are more work to enforce than they're worth.  Actually I would go a step further and say that the pins will all be up for consistency.  That way the scrambler can just immediately lie the puzzle on the table when they're done scrambling.

I disagree with (2).  I think that shaves a reasonably large amount of time.  I also think it makes first-clock skips easier, since you don't need to remember to go to a different pin position than you normally do, and the main reason that skips are hard to deal with is the change in routine.  Finally, I don't like the idea of allowing the competitor to manipulate the puzzle in only certain ways during inspection; I think this will make our lives more difficult in the long run.  The more event-specific rules we have, the more complicated things become for inexperienced judges, and I don't want judges to get confused about which manipulations are allowed during inspection.

**Comment by AaronJamesBlair** [comment]
1) This would definitely save a massive amount of time for scramblers, runners, judges, and competitors. At a large event, something like US Nationals, this would save several minutes of time, allowing for the entire competition to stay on track a little easier. At a small or more localized event, this makes it easier for those judging who aren't competitors or familiar with clock to judge without confusion. I think all of the pins should be up no matter what, for consistency and ease of runner delivery. It speeds up scrambling and the start of the attempt for the competitor. The judge, at the most, would need to push the clock onto the table to push all of the pins up before starting the attempt.

2) I disagree with this due to the complications of ensuring that only pins are moving. While I can't speak for everyone, I know that even if I'm only playing with the pins on a clock, I push them up with my hands in position to start rotating the wheels.  This could make it hard for a judge to tell if only pins are moving.

Sometimes the wheel's aren't fully aligned and block the pins from being pushed all the way in or out. To fix this, you'd have to rotate the wheel slightly to free up the pin or press the pin aggressively, which would cause the wheel to rotate a little. The click from the wheel rotating might cause a judge to think that the competitor rotated a wheel and could lead to a DNF.

Teaching a judge what puzzle movements are allowed and which ones aren't would take up all of the time saved by (1).

**Comment by pedrosino** [comment]
While I think pins at the start are an important part of the puzzle, I wouldn't mind having them all up if it helps judging. I don't see much advantage for scramblers, since it takes just a couple seconds. Most of the time is spent actually scrambling the puzzle, which is something we should think about improving. 

Would 1 mean that I can ask for a specific face to be with pins up? Or would it be "random"?

About 2, I'm not comfortable letting people move pins. If we choose to make them all up, they should stay that way through inspection.

**Comment by KitClement** [comment]
I agree with most of the people here in that (1) would make an impact on the event, but not in a significant enough way that the advantages to judges is worth it.  I'd agree with implementing (1).

I disagree with implementing (2). Being able to choose a pin position is a huge advantage, and I would believe I could shave off somewhere between a quarter and a half a second on solves that have lucky cases with bad pin placement.  This would have a large enough impact on the event that I would prefer to not have this.

I think that the scramble states should just indicate UP/DOWN at the end indicating whether all the pins are up or down on the second side, and that we should ensure that the clock remains in this pin position when the competitor gets the clock.

**Comment by NikiPlacsko** [comment]
(1) At first sight I liked this idea (because it seemed like that it would make the scramblers'/runners'/judges' work easier), but after thinking it through, I have some concerns about it:
- Fist of all, I don't like this idea if the state of the pins could be different for other competitors (so if it would be random that the pins will be up or down) because it can give advantages/disadvantages for competitors. I think if all the pins are up/down, most of the cubers will start their attempt on that side where all the pins are down, so if there is a lucky case, it matters that on which side are the pins up and down. (It's easier to start with a lucky case than remember to a lucky case)
- For the scramblers it would not be easier at all. The difficulty of scrambling the Clock is not that they need to adjust the pins, but the scrambling itself. Adjusting the pins takes 1-2 seconds, so it doesn't matter from their point of view.
- In the point of view of the runners/judges I can imagine 2 scenarios:

a) We will use the same method to transfer the Clock from the scrambling table to the station, so they will be in standing position. In this case in Clocks, which have loose pins, pins can still move to the other side so it can cause that some or all of the pins will still not in the right position.
And no matter that we have random pins or all pins up/down, the Clocks can still fall on the mat and the judges will have no idea that it is the right or the wrong position of the pins.
b) We can say that Clock should be transferred from the scramblers to the stations laid to something. In this case it would be easy to runners/judges, but would cause troubles for competitors during inspection to lift up the puzzle and check the other side without moving the pins. (This also concern mostly the Clocks with loose pins as well)

(2) At first I didn't really like the idea of letting people to make changes on the Clock during inspection and I'm not sure that I like it or not right now. But in my opinion this is the idea which can make easier to organize Clock at a competition.
Pros:
- Runners/judges would not have to be careful with the puzzles and this would speed up the event.
- We could skip all the extra attempts because of fell down Clocks.
- We could skip all the sent back Clocks to the scramblers to check the pins.

Contras:
- It would give advantages to competitors according to the current regulations.
- As the clocks on the Clock can be moved with turning the pins as well it would be more difficult to judge the event and would need much more attention.
  (Most of the Clocks make noise when someone turns them, but there are a few which are really silent and by this Clocks cheating would be easier.)

**Comment by pedrosino** [comment]
I said

> Would 1 mean that I can ask for a specific face to be with pins up? Or would it be "random"?

What I really mean is: "Would I be able to request that my clock is scrambled with a particular side on the front, so that the pins are always up on the same side?" (I'm asking because one side feels a little better on my clock, so I tend to finish solves with it)

Niki has some good points about 2) (no need to check pins). Building on what she said, if we implement 2), we wouldn't need 1), or at least wouldn't have to worry about pin position at all. I think 15 seconds is enough time to set up your pins during inspection, and most solvers wouldn't complain about pins not being the same for everybody at the start if they are allowed to adjust them before starting the solve.

**Comment by CoreySakowski** [comment]
To clarify about the first proposal, I meant that some scrambles would have all pins up, and some scrambles would have all pins down. The direction would depend on what the scramble says (either up or down), and what side the scrambler begins from.

**Comment by HippolyteM** [comment]
Being purely theoretical, I think that adding change 1) without adding change 2) would hardly make sense:
As Corey pointed out, there is already a not so logic way of considering this puzzle in the current regulations, as they do not request the pin to be in a specific state in the end of the solve, but still consider these pins as pieces worth scrambling. Now, if moreover we choose not to scramble the pins, it become really clear that we do not consider them as we would consider any pieces of any other puzzle (the form of the clock itself making this term of "piece" quite hard to apply), and from this point, it could also seem logic that changing the state of a pin would not be a move in the way we usually understand it. Then, why would we forbid the competitor to make this "not-a-move" change? It would be much more like moving a cube layer by less than 45°.

Now, from a solver point of view, the pin IS an important piece. The simple fact that lot of solvers choose where they start depending on the pins more than depending on the hours is quite revealing. And whether change 1) would still make the pin "scrambled" (just they would always be scrambled the same way), change 2) would totally change the approach of the solver, as they would completely focus on the hours, choose the best beginning and be able to anticipate much further.
So if these change are applied, the challenge seems to be totally different, and having old and new times mixed would feel unfair to me. It would so request a full reboot of the clock times and rankings.

From a organizer point of view, change 1) wouldn't change anything, as it was said, given that we would also have to be careful to which side have pins up.
Change 2) would considerably simplify the scramblers and runners life, as it would make any alteration of the pins insignificant (and therefore change 1) quite useless). It would also make all the really bad clock (with pins always moving) less painful for the runners. But, as Aaron said, it could be a nightmare for judges, since it is sometimes almost impossible to determine if a previously badly aligned clock had a change of position, if you don't check the scramble.

Therefore, even if I regret it, I would not be in favor of any of these changes, which would make lot of works for the website and regulations teams without really making organizers lives more comfortable.
(I would have an idea for solving all the clocks issues, but it's drastic and off-topic)

**Comment by pedrosino** [comment]
> (I would have an idea for solving all the clocks issues, but it's drastic and off-topic)

Maybe start a new issue? :) I'd be interested to hear

**Comment by operge** [comment]
I don't really see the point in adding any of these changes. If we create a new way of transporting the clocks to the stations, that might cause more work to get used to, than simply just paying attention to the way we handle clocks now.

I am definitely against 2), as it would be too much of a game changer for the event as a competitor. I see Hippolyte's point, and technically, changing the positions of the pins during inspection is not even a move, but I don't think that the general judges are capable of confirming if the competitor moved pins only or wheels too, so we should not use the "anything goes" principal here, in my opinion.

I am also interested in the drastic changes, as long as they either add the proper penalties or remove the event at all. :P

**Comment by HippolyteM** [comment]
> remove the event at all. :P

That's what I had in mind, sorry it wasn't clear enough.^^

**Comment by viroulep** [comment]
I like the logic behind 2 but I would not want to implement it, because of the significant time advantage it would provide compared to the current Regulations.

I like 1, because it does facilitate the job of the judges, and would avoid Clocks falling before inspection.
I would expect that implementing 1 would mean that for each scramble we put the pins up the same way for every competitor (for the reason @NikiPlacsko mentioned).
As @timreyn said, the scrambler could just put the Clock flat when he's done, and the runner/judge would transport it to the station this way so that the competitor also start the inspection with the Clock being flat.

Niki brings up a good con to this, but I think Clocks with loose pins would fall under [3a2](https://www.worldcubeassociation.org/regulations/#3a2) as normal scrambling would not be possible under this new regulation. And I'd be totally fine with simply rejecting those Clocks :)

**Comment by Randomno** [comment]
Both of these proposals significantly affect the event. I am generally against changes like this just to make organisation easier, but unlike every other poster here I have never organised a comp.

**Comment by J-Rohault-dF** [comment]
Being able to change pins during inspection would be an advantage, but not a huge one compared to other events: it is allowed to turn a cube ~30° during inspection: it can make the solve faster, mostly on small cubes like 2x2, pyraminx or skewb, where the time taken to pick the cube and adjust the hands for fingertricks is a big part of the solve.

**Comment by KitClement** [comment]
> it is allowed to turn a cube ~30° during inspection

Not intentionally.

**Comment by J-Rohault-dF** [comment]
If the puzzle is brought slightly misaligned, we don't have to align it, the regulations just say we "may" do it (https://www.worldcubeassociation.org/regulations/#A3c2). Also, applying moves is forbidden, but a 10° change isn't "technically" a move according to 12a

**Comment by KitClement** [comment]
That allows you to align, not misalign. Nothing in 12a defines if a 10
degree turn is a move or not.

Generally, the interpretation of the regulations based on A3c1 is that
moves cannot be made during inspection. We will almost always turn some
layers a few degrees unintentionally, so usually the limits of misalignment
are used to allow for unintentional turning and realignments, but
intentionally making a partial turn in inspection to gain an advantage
constitutes a move.

**Comment by lgarron** [comment]
I would like us to consider this change for 2020. 

The pin state is not relevant to the solved puzzle and it would be simpler for many competitions if they did not have to worry about pin state during transport.

**Comment by dancing-jules** [comment]
I am also against (2). It would make new records incomparable to old records. 

About UP or DOWN. I think if the WCA implements (1), it should be all UP. The two sides are indistinguishable anyway, so if the scramble said DOWN, the judge would put down the clock on the mat with the backside up, which makes it identical to the mirrored scramble with pin position UP.

Requesting a specific side up (as pedrosino suggested) is in my opinion a very bad idea. That would mean that different people in the same group get a different scramble.

**Comment by DanielEgdal** [comment]
I'm against both of these. To me, option 2 is the least bad though.

Why I dislike 1:

- You would always have to change at least 1 pin to do the first move (unless the scramble is incredibly lucky and you have all edge clocks matching the center). I assume this would add at least 0.2 to all solves making it incredibly hard to beat past results.
- If the clock is lying down, it would require extra time for the competitor to pick it up. If it's standing, it still has the chance of some pins being pushed just as now.

Why I like 2:
- It would make hosting the event easier. The pin position doesn't matter from any organizational view. The competitor is completely in charge of it.
- It can make sure no pins are stuck if the scrambler didn't allign the last move properly (sometimes the internal mechanism messes up slightly which is pretty hard to see in inspection unless you alter the pins).
- (Controversial) All solve times will get faster.

Why I dislike 2:
- It will be harder to teach judges how the event works. Some people might make a move on the internal clocks (maybe by just twisting the pin).
- It will look weird with people doing turns during inspection. With the loud magnet clocks it will sounds suspicious to hear the clicks.

I would (for now) mostly be in favour of leaving the pins how they are handled for now. They matter for the solve with how they are positioned.

**Comment by cubewhiz** [comment]
I think allowing pins to be changed is difficult for judges to understand. I am opposed to allowing competitors to make alterations to the pins during inspection for this reason.

**Comment by alexmaasswca** [comment]
Allowing pins to be changed will actually make clock solves faster though, and yeah on the loud magnet clocks, it will look definitely sound suspicious to hear clock clicking, it would also be possible for the competitor to do a dial move and claim it was just the noise of the pin (a stretch though). 

Also if you keep all the pins up, it will make clock solves slower since you're only really limited to solving on one side since it would be slower to start the solve on the side with all the pins up since you'd have to move 3 pins down at the start of the solve. Also you still have to make sure the pins aren't disturbed during transport.

Why make these changes when significant and successful efforts have been made with special clock covers?

**Comment by lgarron** [comment]
Copying from #856:

@SeanMoran27 at https://github.com/thewca/wca-regulations/issues/856#issuecomment-541311689
> Just wanted to clarify one point on the new regulations:
> 
> Clock pins are all pushed down at the end of the scramble.
> 
> Does this mean we can start clock attempts face down or is it still necessary to have them upright at the start of the attempt?

@Ivan-Brigidano at https://github.com/thewca/wca-regulations/issues/856#issuecomment-541312395
> @SeanMoran27 the idea behind this is exactly having the possibility of start the attempt with the clock facing down to avoid all the problems the upright position has.

@simonkellly at https://github.com/thewca/wca-regulations/issues/856#issuecomment-541316181
> > > Clock pins are all pushed down at the end of the scramble.
> > Pro: This makes it easier to scramble correctly and transport the scrambled puzzle without affecting > the state.
> Would this allow competitors to place the puzzle face down at the end of their inspection period, with their choice of sides facing up? As if the rule was that the puzzle must be face down in any orientation, that would allow almost all precautions regarding not affecting the puzzle during transport

@jamesquinn1 at https://github.com/thewca/wca-regulations/issues/856#issuecomment-541329904
> Clock pins- Requiring clock pins to be all facing the same way and having the clock being facing down on that side is actually harder to manage than face up if you have clock covers (which are becoming more and more common, and cost ~$2 per cover). This makes new results also harder to obtain as the first move of a clock solve is (almost) never an ALL move, but rather a move that is influenced by the pin position.
>
> [...]
>
> Clock pins: Can this be clarified further? What I'm currently getting is that the pins would all be facing one way in the scramble, and they must stay facing that way during transport and inspection, and the competitor could not move them during inspection, but the competitor could stand the clock up. Is this correct?

@UnderwaterCube at https://github.com/thewca/wca-regulations/issues/856#issuecomment-541331995
> For the clock bit you mention that it’s unfair to past competitors, as someone who does clock I can tell you that having all pins starting up is actually pretty bad. It’s like having to solve 1 tip on pyra vs 4 tips on Pyra. The place you have probably seen the “it’s unfair to old competitors” is on discussions where people are proposing that competitors may push in pins in inspection to whatever they would like, that does give new competitors and advantage over old competitors.
> 
> Don’t have time rn but I will drop by later to share my thoughts on some other things.

@UnderwaterCuber inathttps://github.com/thewca/wca-regulations/issues/856#issuecomment-541336887
> Disadvantages new competitors but makes things a lot nicer, an alternative could be to allow competitors to manipulate pins in inspection which would instead disadvantage old competitors.

@Clqw at https://github.com/thewca/wca-regulations/issues/856#issuecomment-541342813
> Make it DNF to move pins before the attempt? Or just accept them moving in inspection?

**Comment by lgarron** [comment]
Responses to some of the questions above:

Yes, part of the benefit of having all pins face up is that the competitor can lay the clock flat without penalty. I don't yet have information on how many clock solvers would *prefer* this option individually, but it certainly makes it easier to to avoid an accident.

In principle, I think it's a pretty reasonable argument that the clock state should completely disregard pins and that competitors should be able to toggle them during inspection. However, I agree with @cubewhiz that this can be difficult for judges to understand — or to observe accurately. So I think we should avoid that change for now.

**Comment by jamesquinn1** [comment]
Thank you for clarifying.

"Yes, part of the benefit of having all pins face up is that the competitor can lay the clock flat without penalty. I don't yet have information on how many clock solvers would prefer this option individually, but it certainly makes it easier to to avoid an accident."
I don't think anyone under 20 seconds would choose this, as it would add time to their solve.

With new clock covers I've found that putting them standing up is quite easy, and requiring them to be facedown in a certain direction would be quite a hassle.

**Comment by Jambrose777** [comment]
I'll chime in here with regards to having all 4 clock pins face in the same direction for every solve.

This change would case the event to become more challenging. The number of possible solutions that a competitor can do is reduced as they are now /mostly/ restricted to 1 side to start, especially in the mediocre level of solving. And it would significantly change the strategies in solving.

Most starting moves require 1 pin to be up on a face. From an all down position, this would mean that you push 1 pin up (which is the easiest/fastest type of 1 pin push) or from the all up positions 3 pins down, which is incredibly solve when followed by a move (requires both thumbs to be used). compared to now, where most beginning pin changes can require 1 thumb press and 1 back press to reach your starting pin position.

This would cause the event to have less unique /good/ solutions per scramble and ultimately cause a bigger affect of most solutions being the same. You would be filtering more scramble states (7/8 of them) in an event that is incredibly scramble dependent. More clearly, I would say that this reduces the number of good scrambles possible for the reasons above.

WRC stated in the OP "This will simplify the work that competitors need to do at the beginning of an attempt. This is not fair to past competitors", however I don't believe that this is the case and that this change would cause a significant disadvantage to clock solvers compared to past competitors.

**Comment by alexmaasswca** [comment]
Also, more recently, more and more competitions have been investing into getting properly specially made clock covers, this would pretty much waste all of that effort as those covers would not have any usage after that.

**Comment by edladd84** [comment]
At my (low) level of Clock expertise, manipulating the pins at the start of a solve is a significant part of the solve.  I'll usually choose my starting side based on how easy it will be to get the pins to my usual first position.

Maybe that is something that doesn't affect faster people.  But I find it hard to imagine that we wouldn't be giving up part of the complexity of the solve, in figuring out how to finger-trick the pin state during the pick-up.
